components/match2/commons/match_summary_base.lua

```diff
@@ -176,10 +176,10 @@ function Footer:addLinks(links)
 		if not currentLinkData then
 			mw.log('Unknown link: ' .. linkType)
 		elseif type(link) == 'table' then
-			Array.forEach(link, function(gameLink, gameIdx)
+			for gameIdx, gameLink in Table.iter.spairs(link) do
```

Comment:
This will break on the 10th link right?

Reply:
```suggestion
			for gameIdx, gameLink in Table.iter.spairs(link, orderLinks) do
```
with
```lua
local orderLinks = function(_, key1, key2)
	local index1 = tonumber((key1:gsub(key1, '%d*$'))) or 0
	local index2 = tonumber((key1:gsub(key2, '%d*$'))) or 0
	return index1 < index2
end
```

Reply:
Keys are numeric (in the lua table, only the json representation in LPDB uses the pseudo arrays), so shouldn't break?
![image](https://github.com/user-attachments/assets/c7672995-cb67-4745-b092-0ba2f5c3d7a4)